gdal/PROVENANCE.TXT

```diff
@@ -429,12 +369,6 @@ I have added this license to LICENSE.TXT to satisfy the credit requirement.
 
 * FrankW, clean.
 
-=== gdal/frmts/hdf ===
-
-* geoextra.c: Derived by public domain contribution to libgeotiff from PCI but the terms are not clear.  This is a libgeotiff issue.  There is no problem with the license - it just isn't well documented in the code file.
-* Mostly copyright Intergraph Corporation, clean.
```

Comment:
This line and the following lines are still valid. Only the one about geoextra.c is no longer current